toolchain/check/handle_operator.cpp

```diff
@@ -399,7 +404,7 @@ static auto HandleShortCircuitOperator(Context& context, Parse::NodeId node_id)
   context.AddInst<SemIR::BranchWithArg>(
       node_id, {.target_id = resume_block_id, .arg_id = rhs_id});
   context.inst_block_stack().Pop();
-  context.AddCurrentCodeBlockToFunction(node_id);
+  context.AddToRegion(context.inst_block_stack().PeekOrAdd(), node_id);
```

Comment:
Isn't this `context.inst_block_stack().PeekOrAdd()` the same as `resume_block_id`? Could we use that directly here?